uber/config.py

```diff
@@ -97,8 +97,10 @@ class Config(_Overridable):
     """
 
     def get_oneday_price(self, dt):
-        default = self.DEFAULT_SINGLE_DAY
-        return self.BADGE_PRICES['single_day'].get(dt.strftime('%A'), default)
+        return self.BADGE_PRICES['single_day'].get(dt.strftime('%A'), self.DEFAULT_SINGLE_DAY)
+
+    def get_presold_oneday_price(self, badge_type):
+        return self.BADGE_PRICES['single_day'].get(self.BADGES[badge_type], self.DEFAULT_SINGLE_DAY)
```

Comment:
We need a separate function for "presold" single day passes, since their cost is based on badge type rather than the date on which they were registered.